src/components/Personal-Zone/Publisher/JobsList/job.tsx

```diff
@@ -16,6 +18,7 @@ interface JobsProps {
 const Job: React.FC<JobsProps> = (props): JSX.Element => {
     const classes = styles();
     const { ad } = props;
+  const [isOpenCloseAdDialog, setIsOpenCloseAdDialog] = useState(false);
```

Comment:
I think `closeAdDialogState` is more understandable, just because the dialog have the word close in it